src/time_zone_info.cc

```diff
@@ -534,9 +534,16 @@ std::unique_ptr<ZoneInfoSource> FuchsiaZoneInfoSource::Open(
   // Prefixes where a Fuchsia component might find zoneinfo files,
   // in descending order of preference.
   const auto kTzdataPrefixes = {
+      // The tzdata from `config-data`.
       "/config/data/tzdata/",
+      // The tzdata bundled in the component's package.
       "/pkg/data/tzdata/",
+      // General data storage.
       "/data/tzdata/",
+      // The recommended path for routed-in tzdata files.
+      // See for details:
+      // https://fuchsia.dev/fuchsia-src/concepts/process/namespaces?hl=en#typical_directory_structure
+      "/config/tzdata/",
```

Comment:
Just checking that these are indeed in "descending order of preference".  Naively, it seems strange that "/config/data/tzdata/" and "/config/tzdata/" are on opposite ends of the spectrum.

Reply:
In a way yes, this is the correct ordering. `/config/data/...` is deprecated, and `/data/tzdata` *may* be present in some older components, and `/pkg/data` should probably take precedence over `/config/tzdata` if the owners so wants.

Reply:
From the way you describe things, it sounds like the ordering should be:

1. `/pkg/data/tzdata/`
2. `/config/tzdata/`
3. `/config/data/tzdata/`
4. `/data/tzdata`

But, it's your choice.